metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -100,6 +101,7 @@ func TestFetch(t *testing.T) {
 }
 
 func TestData(t *testing.T) {
+	t.Skip("flaky")
```

Comment:
This is an interesting one. We started to remove the `compose.EnsureUp` in the data tests as they are skipped anyways later on. Perhaps try to also in your other PR. The idea is that we move this to the new data tests framework. Having the Stack modules there too would be great. Please ping me if you need more details.

Reply:
@ruflin If you could point me to a PR that shows a module moving to the new data tests framework, that would be very helpful. If it's too much trouble, I can look around in the code and try to figure it out too.

Reply:
Here is an example: https://github.com/elastic/beats/pull/11112 But we should probably still sync up as Elasticsearch is a bit different because it needs more then 1 endpoint available which is not supported yet (but we should add it).